Create ui-c8y-1024-13-1-fix-scada-svg-upload-staying-stuck-after-first-upload-12680-graft.md (#5009)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1024-13-1-fix-scada-svg-upload-staying-stuck-after-first-upload-12680-graft.md

---
date: ""

> title: Fix SVG upload being stuck on loading state after the first upload in SCADA widget

Reviewer: ```suggestion
title: Fixed SVG upload issues in SCADA widget
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-67282
version: 1024.13.1
---

> When uploading an SVG file in the SCADA widget, the upload process would become stuck after the first successful upload, preventing you from uploading another file without reopening widget config. This issue has been fixed so that you can now upload another SVG file without encountering any stuck states or needing to reopen widget config.

Reviewer: ```suggestion
When uploading an SVG file in the SCADA widget, the upload process got stuck after the first successful upload, preventing you from uploading another file without reopening the widget configuration. This issue has been fixed. You can now upload another SVG file without encountering any stuck states or having to reopen the widget configuration.
```

---
date: ""

> title: Fix SCADA SVG upload staying stuck after first upload (#12680) [GRAFT][release/cd] (#12792)

Reviewer 2: ```suggestion
title: Fix SVG upload being stuck on loading state after the first upload in SCADA widget
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-67282
version: 1024.13.1
---
# Backport

This will backport the following commits from `develop` to `release/cd`:
- [fix(Cockpit): [MTM-67282] Fix SCADA SVG upload staying stuck after
first upload
(#12680)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/12680)

<!--- Backport version: unknown -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[MTM-67282]:
https://cumulocity.atlassian.net/browse/MTM-67282?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Co-authored-by: Paweł Rynarzewski <92171763+pawel-rynarzewski-c8y@users.noreply.github.com>

> Co-authored-by: Claude Sonnet 5 <noreply@anthropic.com>

Reviewer 2: ```suggestion
When uploading an SVG file in the SCADA widget, the upload process would become stuck after the first successful upload, preventing you from uploading another file without reopening widget config. This issue has been fixed so that you can now upload another SVG file without encountering any stuck states or needing to reopen widget config.
```
